src/types.ts

```diff
@@ -14,6 +14,15 @@ declare global {
     }
 }
 
+declare module 'express' {
+    export interface Request {
+        id: string;
```

Comment:
Should this interface extension be duplicated both for global and express modules? Can we only extend it once?